{"uuid": "83a37888-00f7-4caa-b55d-ec5287b80978", "children": ["e2341bb9-7221-4863-ac1d-dc25118fd972"], "befores": [{"name": "login", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Administrator\\Desktop\\test\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\Administrator\\Desktop\\test\\test_审批流程.py\", line 62, in login\n    raise e\n  File \"C:\\Users\\Administrator\\Desktop\\test\\test_审批流程.py\", line 39, in login\n    captcha_data = captcha_img.get_attribute(\"src\").split(\",\")[1]\n                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^\n"}, "attachments": [{"name": "登录失败截图", "source": "a90ee378-42fa-4703-af21-209364fd0298-attachment.png", "type": "image/png"}], "start": 1751619621326, "stop": 1751619623685}], "start": 1751619621326, "stop": 1751619623728}